CHANGELOG.md

```diff
@@ -2,283 +2,520 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
+## [0.7.5] - 2024-03-12
```

Comment:
This and every other link generated by headers is in this format: https://github.com///compare/v0.7.4..v0.7.5
which isn't correct and results in 404 page not found

Reply:
Fixed https://github.com/3mdeb/meta-rte/compare/59973da3165f4b6227a4a5eb9279c0cb3b63d074..c3d9963d89997d076211040aeac3e5fba9cab212#diff-e1372c8b03c40942b5d828a90975054cb8aaed3b38189f434396f922ec41a584R80